### PR TITLE
Homework: Password check

### PR DESCRIPTION
**Pull request by MalarzDawid**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: password-check
🏅 Granted 8 XP for @MalarzDawid

**Comment by ziobron** [comment]
✅ CI Passed - Task: password-check-bonus
🏅 Granted 4 XP for @MalarzDawid

File: homework/password-check/validation.cpp [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+            return "Passwords do not match";
+    };
+    return "None";
+};
+
+bool doPasswordsMatch(const std::string& password, const std::string& repeated_password) {
+    if (password == repeated_password) {
+        return true;
+    }
+    return false;
```
Comment (nauka-programowania-MA) [inline homework/password-check/validation.cpp:27]: Just `return lhs == rhs`

**Review by nauka-programowania-MA — APPROVED** [review]
Nice! I like it :D

**Pull request closed without merging**